### ISSUE TITLE
MySQL defaults are not applied

### ISSUE THREAD

**Issue by x70b1** [influxdata/telegraf#6444]
### Relevant telegraf.conf:
```toml
[[inputs.mysql]]
  servers = ["user:pass@tcp(127.0.0.1:3306)/?tls=false"]

  metric_version = 2
#  gather_table_schema = true
#  gather_process_list = true
#  gather_user_statistics = true
#  gather_info_schema_auto_inc = true
#  gather_innodb_metrics = true
#  gather_slave_status = true
#  gather_binary_logs = false
#  gather_table_io_waits = true
#  gather_table_lock_waits = true
#  gather_index_io_waits = true
#  gather_event_waits = true
#  gather_file_events_stats = false
#  gather_perf_events_statements = false
#  interval_slow = "10m"

  [inputs.mysql.tags]
    influxdb_database = "Mysql"

```

### System info:

Telegraf 1.12.0 in Docker

### Steps to reproduce:

1. use Telegraf with config from above
2. verify the collected data
3. there is only a `mysql` measurement

### Expected behavior:

I would expect that the values from the example config are already the default values and don't have to be set explicitly.

If I set in my config `gather_user_statistics = true` Telegraf collects the data.

### Actual behavior:

It seems that the default values are not applied in the MySQL plugin. So either the plugin has a bug or the README of the mysql input plugin should be modified.

**Comment by danielnelson** [comment]
Thanks for reporting this bug, we should fix the README/SampleConfig.

**Comment by danielnelson** [comment]
Because I needed to modify some default values I think the fix will be more appropriate for the 1.13.0 release.
